index.html

```diff
@@ -497,8 +497,8 @@ <h2>Content</h2>
 <li><a style="word-break: break-word" href="br_ibge_README.txt">br_ibge_README.txt</a>. Last modified: 2020-06-29</li>
 <li><a style="word-break: break-word" href="br_ibge_CA61_003.tif">br_ibge_CA61_003.tif</a> - Brazil - Corrego Alegre 1961 - Corrego Alegre 1961 (EPSG:5524) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
 <li><a style="word-break: break-word" href="br_ibge_CA7072_003.tif">br_ibge_CA7072_003.tif</a> - Brazil - Corrego Alegre 1970-1972 - Corrego Alegre 1970-72 (EPSG:4225) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
-<li><a style="word-break: break-word" href="br_ibge_SAD69_003.tif">br_ibge_SAD69_003.tif</a> - Brazil - onshore south of 14°S and east of 53°W - SAD69 (EPSG:4618) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
-<li><a style="word-break: break-word" href="br_ibge_SAD96_003.tif">br_ibge_SAD96_003.tif</a> - Brazil - onshore south of 14°S and east of 53°W - SAD69(96) (EPSG:5527) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
+<li><a style="word-break: break-word" href="br_ibge_SAD69_003.tif">br_ibge_SAD69_003.tif</a> - Brazil - onshore south of 14�S and east of 53�W - SAD69 (EPSG:4618) to SIRGAS 2000 (EPSG:4674). Last modified: 2020-06-29</li>
```

Comment:
Another encoding change. But apparently in the other direction.

Reply:
I have done a new regeneration of index